objects/Script/Apply deployment package settings for automatic deployment rule.ps1

```diff
@@ -0,0 +1,35 @@
+﻿<# Apply-ADRDeploymentPackageSettings #>
+
+#=============================================
+# START SCRIPT
+#=============================================
+param
+(
+[parameter(Mandatory = $true)]
+[ValidateNotNullOrEmpty()]
+[string]$sourceADRName,
+
+[parameter(Mandatory = $true)]
+[ValidateNotNullOrEmpty()]
+[string]$targetADRName
+)
+
+Try {
+       # Source ADR that already has the needed deployment package. You may need to create one if it doesn’t exist.
+       $sourceADR = Get-CMSoftwareUpdateAutoDeploymentRule -Name $sourceADRName

```

Comment:
Before using $sourceADRName and $targetADRName, you should consider validating that they are the correct size (<= whatever the max is for an ADR name).